### ISSUE TITLE
Enable changing of corner roundness

### ISSUE THREAD

**Issue by benrap** [Maddoc42/Android-Material-Icon-Generator#14]
How about an option to enable changing of corner roundness when choosing the square background?

**Comment by Maddoc42** [comment]
Yes, that would be great :+1: in fact, this could even replace the static option for square / round altogether and let the corner radius define how round the background is :rocket:

Implementation should be "fairly" straight forward, roughly:
- [ ] Add slider UI similar to shadow length etc.
- [ ] Add listener to slider for reading corner radius. Values are between 0% and 100%, where 100% equals half the length of the background square which will create a perfect circle.
- [ ] From the listener update the background radius, making sure to rerender the shadow after each update (see other sliders as an example).
- [ ] Delete 'old' background UI option and remove associated code.

Am I missing something? :memo:
